### PR TITLE
implemented throwing of exception in managers commit

### LINKED ISSUE #648: Throw exception on document commit error
Response from `$manager->commit()` is returned, but not checked for errors. Problems are hard to track, especially when new documents violate the mapping silently.

This is debatable, but generally, I vote in favour for throwing a custom exception by default. Cases are rare, when one would like to ignore such error and this could be easily configured using a parameter or try-catch block.

I strongly suggest, that exception must be thrown by default especially in debug environments.

### PR DESCRIPTION
**Pull request by einorler**
Closes #648

### CODE REVIEW

File: Service/Manager.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 namespace ONGR\ElasticsearchBundle\Service;
 
 use Elasticsearch\Client;
+use Elasticsearch\Common\Exceptions\ClientErrorResponseException;
```
Comment (saimaz) [inline Service/Manager.php]: Should be `use Elasticsearch\Exception\ClientErrorResponseException;`

File: Service/Manager.php [hunk 2]
```diff
@@ -337,6 +340,12 @@ public function commit(array $params = [])
                     break;
             }
 
+            if ($bulkResponse['errors']) {
+                throw new ClientErrorResponseException(
```
Comment (saimaz) [inline Service/Manager.php]: I think we could pass also the whole bulk response to the exception message. No?

Comment (einorler) [inline Service/Manager.php]: Do you mean a json serialized bulk response array?

File: Service/Manager.php (hunk 1 above)
Comment (einorler) [inline Service/Manager.php]: No class with this namespace can be found

**Comment by saimaz** [comment]
Please rebase PR

**Pull request merged**